bec_widgets/widgets/utility/logpanel/logpanel.py

```diff
@@ -11,10 +11,10 @@
 from typing import TYPE_CHECKING, Literal
 
 from bec_lib.client import BECClient
-from bec_lib.connector import ConnectorBase
 from bec_lib.endpoints import MessageEndpoints
 from bec_lib.logger import LogLevel, bec_logger
 from bec_lib.messages import LogMessage, StatusMessage
+from pyqtgraph import SignalProxy
 from PySide6.QtCore import QObject
```

Comment:
Please change to `qtpy`

```suggestion
from qtpy.QtCore import QObject
```